mithril-aggregator/src/configuration.rs

```diff
@@ -121,6 +121,13 @@ pub struct Configuration {
     /// Specific parameters when [snapshot_compression_algorithm][Self::snapshot_compression_algorithm]
     /// is set to [zstandard][CompressionAlgorithm::Zstandard].
     pub zstandard_parameters: Option<ZstandardCompressionParameters>,
+
+    /// Url to CExplorer list of pools that will be pulled to import the pools as signer in
+    /// the database (including their pool ticker).
+    pub cexplorer_pools_url: Option<String>,
```

Comment:
Do we want to keep that option with a more generic name such as `external_pools_url`?

Reply:
The list fetched here must follow the cexplorer format so I think that being explicit is needed.